rmw_connext_shared_cpp/src/shared_functions.cpp

```diff
@@ -555,6 +574,8 @@ destroy_node(const char * implementation_identifier, rmw_node_t * node)
   node->data = nullptr;
   rmw_free(const_cast<char *>(node->name));
   node->name = nullptr;
+  rmw_free(const_cast<char *>(node->namespace_));
```

Comment:
`node_handle` is just the variable name in the `create_node` function. In this function, `destroy_node` that same object is passed in as `node` rather than `node_handle`. At least that's how I read the code.